### PR TITLE
Add altinn log

### LINKED ISSUE #247: Log unsuccessful tries for sending AGA/FTR invoices
Whenever the sending of AGA/FTR invoice to fakturabank fails we should log all possible info to a log somewhere.

### PR DESCRIPTION
**Pull request by mladjo2505**
Closes #247

### CODE REVIEW

**Comment by mladjo2505** [comment]
Some of this needs to be translated into norwegian.

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

As I understood @mladjo2505, it is required to make page which will display all failed attempts to send AGA and FTR invoice. 
Table is made so it can be used for some other failure logging. 

There are translations that are not finished. To be done with @stoivo.

File: modules/altinnsalary/view/log.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+<body>
+
+<? includeinc('top') ?>
+<? includeinc('left') ?>
+<? print $_lib['message']->get() ?>
+
+<table class="lodo_data">
+<thead>
+  <tr>
+    <th colspan="7">Altinn log:</th>
```
Comment (stoivo) [inline modules/altinnsalary/view/log.php]: Altinn logg

File: modules/altinnsalary/view/log.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<?
+print $_lib['sess']->doctype ?>
+
+<head>
+    <title>Altinn log</title>
```
Comment (mladjo2505) [inline modules/altinnsalary/view/log.php]: Altinn logg

File: modules/altinnsalary/view/log.php (hunk 1 above)
Comment (mladjo2505) [inline modules/altinnsalary/view/log.php]: logg

File: modules/altinnsalary/view/log.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+<? print $_lib['message']->get() ?>
+
+<table class="lodo_data">
+<thead>
+  <tr>
+    <th colspan="7">Altinn log:</th>
+  </tr>
+  <tr>
+    <td class="menu">ID</td>
+    <td class="menu">TS</td>
```
Comment (stoivo) [inline modules/altinnsalary/view/log.php]: Tidspunkt

File: modules/altinnsalary/view/log.php [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    <td class="menu">Type</td>
+    <td class="menu">Class</td>
+    <td class="menu">Message</td>
```
Comment (stoivo) [inline modules/altinnsalary/view/log.php]: Beskrivelse

File: modules/altinnsalary/view/log.php [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    <td class="menu">PersonID</td>
+    <td class="menu">AltinnReference</td>
```
Comment (stoivo) [inline modules/altinnsalary/view/log.php]: Altinnreferanse

File: modules/altinnsalary/view/log.php [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  <tr>
+    <th colspan="7">Altinn log:</th>
+  </tr>
+  <tr>
+    <td class="menu">ID</td>
+    <td class="menu">TS</td>
+    <td class="menu">Type</td>
+    <td class="menu">Class</td>
+    <td class="menu">Message</td>
+    <td class="menu">PersonID</td>
```
Comment (stoivo) [inline modules/altinnsalary/view/log.php]: Brukernavn

**Pull request merged**